pkg/securitycontext/provider_test.go

```diff
@@ -259,6 +288,56 @@ func TestModifySecurityOption(t *testing.T) {
 	}
 }
 
+func TestDetermineEffectiveSecurityContext(t *testing.T) {
+	fullySpecifiedSecurityContext := securityContextWith(boolPtr(true), intPtr(1), boolPtr(true), boolPtr(true))
```

Comment:
This is the kind of thing that makes me wish golang had named parameter passing...